packages/simple-types/test/dashboard.test.ts

```diff
@@ -125,6 +126,36 @@ describe("Module `dashboard`: manages the creation and deployment of dashboard i
     }
   });
 
+  describe("_getDatasourceDependencies", () => {
+    it("handles defaulting to .itemId", () => {
+      const obj: any = {
+        dataSource: {
+          itemId: "{{934a9ef8efa7448fa8ddf7b13cef0240.layer0.itemId}}"
+        },
+        datasets: [
+          {
+            dataSource: {
+              itemId: "AAABBBCCC123"
+            },
+            type: "serviceDataset"
+          },
+          {
+            dataSource: {
+              itemId: "AAABBBCCC123"
+            },
+            type: "serviceDataset"
+          }
+        ]
+      };
+
+      const itemTemplate: common.IItemTemplate = templates.getItemTemplate(
+        "Dashboard"
+      );
+
+      dashboard._getDatasourceDependencies(obj, itemTemplate);
```

Comment:
@MikeTschudi should these have some sort of expect at the end or ok with the fact that they simply complete?